### ISSUE TITLE
Video buffering 

### ISSUE THREAD

**Issue by pathania07** [yuliskov/SmartTube#3030]
Many videos on the smarttube app just keeps buffering after some time of playback. My internet speed is good. But the buffering is just making smartTube unusable. With buffering set to low it lags and also with none

**Comment by nbi1** [comment]
Agreed. The buffering has reached a level at which SmartTube is unusable. I'm at version 19.91 and this problem started about 3 months ago which was 2-3 updates ago? At first I thought it was my Internet connection or my WiFi setup, but I have done rigorous testing and my infrastructure is not the problem. Other apps play streaming sources just fine (Prime Video, Philo, YouTube, etc.) so this is definitely a SmartTube issue. I also noticed peculiar error messages during playback of some videos, specifically OOM (out of memory?). When I play the same videos with the YouTube app no errors are generated. Also other streaming apps do not produce these errors.

**Comment by cgiovenco** [comment]
Same, been having lot of issues in the last couple months with buffering/freezing. Have no issues using Kodi or other native apps on my firestick. It seems just related to smarttube. I tried adjusting the network stacks without any noticeable improvement. Hopefully they can get the stability back to where it used to be.

**Comment by Hexinator1302** [comment]
This has also happened to me a lot recently. If I then open the same video on the original app, it runs through without buffering. I have also changed the buffer size, no difference.

**Comment by gary555** [comment]
I also have a buffering problem - I can't watch anything anymore!

**Comment by JReming85** [comment]
You can try changing the video preset, till it works again.

I noticed if you just want to listen you can turn off the video on the preset and it starts playing immediately.

Otherwise you can try switching them to find one that works.  Huge pia
